extensions/amp-subscriptions/0.1/test/test-crypto-handler.js

```diff
@@ -44,15 +44,14 @@ describes.realWin(
     // eslint-disable-next-line max-len
     const decryptedContent =
       "\n              This is section is top secret.\n              You should only be able to read this if you have the correct permissions.\n              If you don't have the correct permissions, you shouldn't be able to read this section at all.\n            ";
-    // eslint-disable-next-line max-len
-    const encryptedKey =
-      "ENCRYPT({'AccessRequirements': ['googleAccessRequirements:123'], 'Key':'mSfq5tRx5omXoOX20Oqq8g=='})";
+    const encryptedKeyBase64 = 'Oik/+==';
```

Comment:
Please add a comment saying what this is in plaintext.

Reply:
Added 👍